client/src/views/NewProject.vue

```diff
@@ -105,17 +106,22 @@ export default {
 };
 </script>
 
-<!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
 p,
 h2,
 ul {
   color: white;
 }
+.margin-bottom {
+  margin-bottom: 15px;
```

Comment:
This goes for everywhere else below as well. Don't use static sizes like px but instead make them dynamic by using vh (view height) or vw (view width).